docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+services:
+  # TODO: Add the Dockerfile for the FAISS Index service
+
+  api:
```

Comment:
Do you also need a data service? 

The following will allow the container to use the GPU which is likely needed for the data  service.
```
    deploy:
      resources:
        reservations:
          devices:
            - driver: nvidia
              count: "all"
              capabilities: [gpu]
```